frappe/tests/test_query.py

```diff
@@ -202,6 +202,20 @@ def test_implicit_join_query(self):
 			),
 		)
 
+		self.assertEqual(
+			frappe.qb.engine.get_query(
+				"Note",
+				filters={"name": "Test Note Title"},
+				fields=["name", "`tabNote Seen By`.`user` as seen_by", "`tabNote Seen By`.`idx` as idx"],
+			).run(as_dict=1),
+			frappe.get_list(
+				"Note",
+				filters={"name": "Test Note Title"},
+				fields=["name", "`tabNote Seen By`.`user` as seen_by",  "`tabNote Seen By`.`idx` as idx"],
```

Comment:
This isn't formatted... @Aradhya-Tripathi setup pre-commit? Why we have to waste time on this again and again :facepalm: